source/Distribution.cpp

```diff
@@ -0,0 +1,75 @@
+/* Distribution.cpp
+Copyright (c) 2022 by DeBlister
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+*/
+
+#include "Distribution.h"
+
+#include "Random.h"
+
+#include <vector>
+
+namespace {
+	double ManipulateNormal(double smoothness, bool invert)
+	{
+		// Center values within [0, 1] so that fractional retention begins to accumulate
+		// at the endpoints (rather than at the center) of the distribution.
+		double randomFactor = Random::BMNormal(0.5, smoothness);
```

Comment:
I would like you to remove this `ManipulateNormal` method entirely, by sampling the random numbers from the target distribution shape. If that means you need to use a not-quite-gaussian in order to have an invertible cumulative probability distribution function, then so be it.

The purpose of the `Random` class is to generate random numbers that obey a given distribution. You should not need to do lengthy processing with the result of calling a `Random::___` method before you can use it for the intended purpose, otherwise the class responsibilities are not correct.